test/authn/Authn.test.ts

```diff
@@ -43,7 +43,8 @@ describe('authn', () => {
 
   it('round trips safely', async () => {
     const originalToken = await authenticator.createToken()
-    const newToken = Token.fromBytes(originalToken.toBytes())
+    const bytes = originalToken.toBytes()
```

Comment:
I've used this to capture the bytes, but it's not really necessary, now that I've learned how to use the VC's debug console :).